### PR TITLE
Fix parsing clusterwide config with empty roles list

### LINKED ISSUE #449: Cluster-wide config with empty roles causes error
See https://github.com/tarantool/cartridge-cli/pull/444#discussion_r559760155

### PR DESCRIPTION
**Pull request by dokshina**
I didn't forget about

- [x] Tests
- [x] Changelog

Closes #449

### CODE REVIEW

File: cli/admin/list.go [hunk 1]
```diff
@@ -2,78 +2,49 @@ package admin
...
+	return &funcInfos, nil
 }
 
 var (
 	adminListFuncBodyTmpl = `
-	local func_help, err = {{ .AdminListFuncName }}('{{ .FuncName }}')
-	return func_help, err
+local func_list, err = {{ .AdminListFuncName }}(...)
+assert(err == nil, err)
+return func_help
```
Comment (mRrvz) [inline cli/admin/list.go:48]: Didn't we add a static filesystem to avoid this (same above)?

Comment (dokshina) [reply cli/admin/list.go]: Yes, now all such Lua code strings can be moved into separated files, but I sure we can perform it in a separated patch.
Added issue https://github.com/tarantool/cartridge-cli/issues/451

File: cli/admin/call.go [hunk 2]
```diff
@@ -36,58 +33,94 @@ type FuncCallArg struct {
...
+	// return res
+	// return nil, err
+	if len(callResData) < 1 || len(callResData) > 2 {
+		return fmt.Errorf("Bad data len: %d", len(callResData))
+	}
+
+	if len(callResData) == 2 {
+		funcErr := callResData[1]
+		if funcErr != nil {
+			return fmt.Errorf("Failed to call %q: %s", funcName, funcErr)
```
[the remark this answers is not in the document]
Comment (dokshina) [reply cli/admin/call.go:63]: Fixed in https://github.com/tarantool/cartridge-cli/pull/444

[the remark this answers is not in the document]
Comment (dokshina) [reply cli/admin/call.go:63]: Fixed in #444

**Review by olegrok — APPROVED** [review]

**Pull request merged**